pyrealm/pmodel/functions.py

```diff
@@ -60,18 +62,19 @@ def calc_ftemp_arrh(
 
     Examples:
         >>> # Relative rate change from 25 to 10 degrees Celsius (percent change)
-        >>> round((1.0-calc_ftemp_arrh( 283.15, 100000)) * 100, 4)
-        np.float64(88.1991)
+        >>> np.round((1.0-calc_ftemp_arrh( 283.15, 100000)) * 100, 4)
+        array([88.1991])
```

Comment:
Python `round` and numpy `np.round` use different rounding strategies. Python `round` uses bankers rounding whereas `np.round` uses round away from zero. This means that under certain conditions they will give different results i.e. one will round up and the other down. This probably doesn't matter too much in this instance, but its worthier keeping in mind.